azure-spring-boot-starters/azure-keyvault-secrets-spring-boot-starter/README.md

```diff
@@ -96,4 +96,24 @@ azure.keyvault.allow.telemetry=false
 When telemetry is enabled, an HTTP request will be sent to URL `https://dc.services.visualstudio.com/v2/track`. So please make sure it's not blocked by your firewall.    
 Find more information about Azure Service Privacy Statement, please check [Microsoft Online Services Privacy Statement](https://www.microsoft.com/en-us/privacystatement/OnlineServices/Default.aspx). 
 
+## Multiple Key Vault support
```

Comment:
What if the user also sets the `azure.keyvault.uri`, what's the order between them?

Reply:
The order would be dependent on the order in the file, but I feel we should caution a customer against doing that. If they opt for multiple key vaults they should only use it that way so the order is guaranteed. I will add this to the documentation.